.github/workflows/upload-deb-images.yml

```diff
@@ -0,0 +1,56 @@
+name: Upload package creation Docker images - DEB - amd64 and i386
+on:
+  pull_request:
+    paths:
+      - 'debs/Debian/**'
+      - 'debs/build.sh'
```

Comment:
Should we include `gen_permission.sh` and `generate_debian_package.sh`?

Reply:
No, they are only used to create the package, not to build the image